src/main/java/net/dries007/tfc/objects/blocks/BlocksTFC.java

```diff
@@ -56,6 +60,114 @@
     @GameRegistry.ObjectHolder("ceramics/fired/large_vessel")
     public static final BlockLargeVessel FIRED_LARGE_VESSEL = getNull();
 
+    @GameRegistry.ObjectHolder("aggregate")
+    public static final BlockGravel AGGREGATE = getNull();
+
+    @GameRegistry.ObjectHolder("alabaster/bricks/white")
```

Comment:
This is not the example of what's been done in other places in TFC, but I think it's both the best solution for this particular case, and what's most like the `RegistryObject` map based stuff I think we should move to in 1.15. So instead:
```java
public static final Map<EnumDyeColor, BlockDecorativeStone> ALABASTER_BRICKS = new HashMap<>();
public static final Map<EnumDyeColor, BlockDecorativeStone> ALABASTER_POLISHED = new HashMap<>();
public static final Map<EnumDyeColor, BlockDecorativeStone> ALABASTER_RAW = new HashMap<>();
```